autogpt_platform/.devcontainer/scripts/poststart.sh

```
@@ -0,0 +1,43 @@
+# =============================================================================
+# This script runs when:
+# 1. Codespace is first created (after postcreate)
+# 2. Codespace resumes from stopped state
+# 3. Codespace rebuilds
+#
+# It ensures services are running without doing heavy setup work.
+# =============================================================================
+
+echo "🔄 Starting services..."
+
+cd /workspaces/AutoGPT/autogpt_platform
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)